app/celery/tasks.py

```diff
@@ -624,3 +624,28 @@ def process_returned_letters_list(notification_references):
             updated, updated_history, len(notification_references)
         )
     )
+
+
+@notify_celery.task(name='send-notify-no-reply')
+@statsd(namespace="tasks")
+def send_notify_no_reply(data):
+    payload = json.loads(data)
+
+    service = dao_fetch_service_by_id(current_app.config['NOTIFY_SERVICE_ID'])
+    template = dao_get_template_by_id(current_app.config['NO_REPLY_TEMPLATE_ID'])
+
+    saved_notification = persist_notification(
+        template_id=template.id,
+        template_version=template.version,
+        recipient=payload["sender"],
+        service=service,
+        personalisation={
+            'sending_email_address': payload["recipients"][0]
```

Comment:
Special case, if people write to `foo@notification.canada.ca` and `bar@notification.canada.ca` at the same time (like multiple fields in `To`/`Cc`/`Bcc`) so always passing a list and using the first argument because this is just really weird.

The list will be logged on CloudWatch though, that's interesting to know that folks are doing this.